documentation/docs/15-configuration.md

```diff
@@ -40,7 +40,8 @@ const config = {
 			params: 'src/params',
 			routes: 'src/routes',
 			serviceWorker: 'src/service-worker',
-			template: 'src/app.html'
+			template: 'src/app.html',
+			errorPage: 'src/error.html'
```

Comment:
`template` and `errorPage` feel like weird siblings, especially now that `errorPage` is _also_ a template. I wonder if this should become `appTemplate` and `errorTemplate`? Wouldn't be a breaking change for most people since most people don't set these values anyway

Reply:
Yeah I was wondering this too, appTemplate/errorTemplate feel more consistent.